### DISCUSSION TITLE
html validation

### DISCUSSION THREAD

**Discussion by NazarAli** [filamentphp/filament#8244] [Help]
hi everyone 👋
how can i disable or remove the html  validation attributes from input form
like required 

![Untitled](https://github.com/filamentphp/filament/assets/7056487/059aa767-38fa-4d90-9102-54a16160a606)

**Comment by leandrocfe** [comment]
you can use `->rule('required')` instead of `->required()`

**Reply by NazarAli** [reply]
> you can use `->rule('required')` instead of `->required()`

thank it work but the required red star not show 🤔

**Reply by leandrocfe** [reply]
Try `->markAsRequired(true)`
